_docs/yasnippets/README.md

```diff
@@ -0,0 +1,21 @@
+# Yasnippets Collection
+This folder contains a yasnippets collection with common templates to ease the development. Yasnippets can be integrated into Emacs: https://github.com/joaotavora/yasnippet .
+
+The snippets can also be used to provide coding guidelines and ensure a general look and feel.
+
+## Quickstart
+Copy the folders in this directory into your local yasnippets emacs configuration.
+
+If you use spacemacs they have to be placed here:
+* ` ~/.emacs.d/private/snippets`.
```

Comment:
`~/.emacs.d/private/snippets` did not worked for me. After installing yasnippet via `M x package-install RET yasnippet` I had to put the snippets to `~/.emacs.d/snippets`.

Alternatively you maybe should add a hint to the docu. that the yasnippet-folder can be adjusted via emacs config
```
(setq yas-snippet-dirs
      '("~/.emacs.d/private/snippets"))   ;; personal snippets
(yas-global-mode 1) ;;
```

Reply:
This only works if you use spacemacs like it is stated in the text.

Reply:
Right, I didn't saw the hint.

Reply:
Maybe you just add a hint for non spacemacs users to put it to `~/.emacs.d/snippets`

Reply:
Solved with: 98676aecedb3d6a9820c444b650f7910a8ff9dea.